spec/tweet_generator_spec.rb

```diff
@@ -1,15 +1,15 @@
-require 'cute_pets/tweet_generator'
+require 'cute_pets/post_generator'
 require 'minitest/autorun'
 
 describe 'TweetGenerator' do
   describe '.create_message' do
     it 'returns a string with the pet name, description and shelter link' do
-      message = TweetGenerator.create_message('Moofle', 'neutered female lab', 'http://www.example.org/moofle')
+      message = PostGenerator.create_message('Moofle', 'neutered female lab', 'http://www.example.org/moofle')
       message.must_match /Moofle. I am a neutered female lab. http\:\/\/www\.example\.org\/moofle/
     end
 
     it 'returns a string with the appropriate article' do
-      message = TweetGenerator.create_message('Miffle', 'unaltered female kitten', 'http://www.example.org/miffle')
+      message = PostGenerator.create_message('Miffle', 'unaltered female kitten', 'http://www.example.org/miffle')
```

Comment:
Line is too long. [114/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.